about.html

```diff
@@ -12,53 +12,111 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    <div class="container">
+
+        <header>
+
+            <nav> 
+
+            <a href="index.html">
+                    <img class="logo" src="img/lambda-black.png" alt="Lambda! Company logo."> </a>
+
+            <a href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="#">Contact</a>
+        </nav>
+
+            
+    </header>
+        
+
+        <div class="services-content">
+
+            <img class="top-img" src="img/jumbo-about.png" alt="top image">
+
+
+        </div>
+        
+
+    <div class="container-about-page">
+      <div class="top-content-p2">
+        
+     
+      <h1>About</h1>
 
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+ 
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <div class="collection-content">
 
-    Learn More
+      <div class="collection-middle">
 
 
+      <img src="img/about-plan.png" alt="strategy">
+      
+      <h1>Strategy</h1>
+      
+      <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+      <div>
+          <button class="about-button">Learn More</button>
+      </div>
 
-    Learn More
+      </div>
 
+      <div class="collection-middle">
```

Comment:
Great job using divs to group your content! I would look into using more semantic elements like "section," but the divs are great for layout!